public/css/menFashion.css

```diff
@@ -0,0 +1,141 @@
+body{
+margin: 0px;
+padding: 0px;
+height: 100vh;
+width: 100%;
+}
+
+.container{
+  width: 1140px;
+  flex-wrap: wrap;
+
+}
+.containernav{
+  display: flex;
+  width: 1140px;
+  align-items: center;
+  justify-content: space-between;
+  flex-wrap: wrap;
+
+}
+.bigContainer{
+  display: flex;
+  flex-direction: column;
+    flex-wrap: wrap;
+}
+header{
+height: 40px;
+background-color: #752223;
+display: flex;
+justify-content: flex-end;
+}
+.headerForm{
+display: flex;
+align-items: center;
+align: left;
+margin-right: 10px;
+}
+input{
```

Comment:
I think you should give this a class name and every thing you make a style for it like as footer,header